docs/data/joy/customization/dark-mode/DarkModeByDefault.js

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { CssVarsProvider } from '@mui/joy/styles';
+import Sheet from '@mui/joy/Sheet';
+import Chip from '@mui/joy/Chip';
+import Typography from '@mui/joy/Typography';
+
+const useEnhancedEffect =
+  typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;
+
+export default function ButtonThemes() {
+  // the `node` is used for attaching CSS variables to this demo, you might not need it in your app.
+  const [node, setNode] = React.useState(null);
+  useEnhancedEffect(() => {
+    setNode(document.getElementById('dark-mode-by-default'));
+  }, []);
```

Comment:
**Off topic**

This could suggest that nesting dark with a light theme in Joy UI, might be a bit of a struggle (it's not a single rendering step).